lib/LicenseTermsList/tests/LicenseTermsList-test.js

```diff
@@ -102,10 +103,18 @@ describe('LicenseTermsList', () => {
     expect(licenseTermsList.licenseTerm(0).value).to.equal(get(license, ['customProperties', terms[0].name, '0', 'value']));
   });
 
-  it('renders the expected note under the first license term', () => {
+  it('renders the expected visibility under the first license term', () => {
+    expect(licenseTermsList.licenseTerm(0).internal).to.equal(get(license, ['customProperties', terms[0].name, '0', 'internal']));
```

Comment:
I don't see the `internal` property defined in the interactor file. Am I missing something?